articles/azure/azs-how-create-scale-set.md

```diff
@@ -0,0 +1,126 @@
+---
+title: How to Create a Scale Set using UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides help for creating a scale set on UKCloud for Microsoft Azure
+services: azure-stack
+author: Shaion O'Niel
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a Scale Set - Portal
```

Comment:
Fix capitalisation:
Create a scale set - Portal

Also, unless there are going to be other articles explaining how do create a scale set using different tools, you don't actually need the portal bit here, so could just change it to:

Create a scale set

Reply:
Unsure if and when another scale set document will be written, removing "- portal".